2.2/alpine3.9/amd64/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-alpine3.9
```

Comment:
Is the only difference here the base image? If so, can we just have 1 of these?

Reply:
That is correct. However, I'm keeping them as separate files so that anyone can just grab the one for their base image and run with it. This is similar to how https://github.com/dotnet/dotnet-docker maintains its Dockerfiles.